docs/architecture.md

```diff
@@ -94,7 +94,7 @@ Distributors are **stateless** and can be scaled up and down as needed.
 
 The distributor features a **High Availability (HA) Tracker**. When enabled, the distributor deduplicates incoming samples from redundant Prometheus servers. This allows you to have multiple HA replicas of the same Prometheus servers, writing the same series to Cortex and then deduplicate these series in the Cortex distributor.
 
-The HA Tracker deduplicates incoming samples based on a cluster and replica label. The cluster label uniquely identifies the cluster of redundant Prometheus servers for a given tenant, while the replica label uniquely identifies the replica within the Prometheus cluster. Incoming samples are considered duplicated - and thus dropped - if received by any replica which is not the current primary within a cluster.
+The HA Tracker deduplicates incoming samples based on a cluster and replica label. The cluster label uniquely identifies the cluster of redundant Prometheus servers for a given tenant, while the replica label uniquely identifies the replica within the Prometheus cluster. Incoming samples are considered duplicated &mdash; and thus dropped &mdash; if received by any replica which is not the current primary within a cluster.
```

Comment:
Looks weird using HTML code (`&mdash;`) in the markdown. Any specific reason?

I would personally not use `&mdash;`, considering this just introduce an inconsistency in the doc.

Reply:
I wanted to fix incorrect usage of hyphens (used as in-word separators) to correct em dashes here. I could have replaced - with – (on Mac, that's option+minus key), but that isn't very explicit in the markdown version. Using HTML markup and entities in Markdown files is a common practice, and Markdown syntax explicitly allows that.

You're right that it's inconsistent with other files, but it seems little too "aggressive" to fix incorrect hyphens everywhere. As I've read architecture doc earlier today, seeing those hyphens where dashes were supposed to be, was just distracting.

Reply:
Not sure the change is worth to be honest, having all such `&mdash;` around just to replace - with –. I would personally rollback such change, but I will let other people comment on it.

Reply:
I tend to agree. It's a super minor improvement imo.

Reply:
If you don't notice these typography bugs, I can understand that.

For me it was just obviously wrong when I read the document, and wanted to fix it.